llm-gemini.el

```diff
@@ -65,7 +65,6 @@ You can get this at https://makersuite.google.com/app/apikey."
   (let ((buf (current-buffer)))
     (llm-request-plz-async (llm-gemini--embedding-url provider)
                            :data (llm-gemini--embedding-request provider string)
-                           :media-type '(application/json)
```

Comment:
The plz-media-types alist already has an entry to handle application/json, so we don't need to add this. It was also an invalid entry. To add or override a media type with the option I think this should have been `(application/json . SOME-MEDIA-TYPE-INSTANCE)`.